gem/models.py

```diff
@@ -68,6 +68,13 @@ class GemSettings(BaseSetting):
                   " line a break. Use only lowercase letters for keywords."
     )
 
+    moderator_name = models.TextField(
+        verbose_name='Moderator Name',
+        null=True,
+        blank=True,
+        help_text="Moderator name"
```

Comment:
@Ntu2koTrevor can we change the help text to `This is the name that will appear on the front end when a moderator responds to a user`